### PR TITLE
Command line to install fonts

### PR DESCRIPTION
**Pull request by 0ctagon**
Now the user just have to do `plothist_install_fonts` in his terminal (Linux or MacOS) to install plothist fonts.

### CODE REVIEW

**Comment by 0ctagon** [comment]
The command line has to actually be checked manually. The function works, but idk for the command line in the pyproject.toml.

**Review by cyrraz — CHANGES_REQUESTED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -27,3 +27,6 @@ dependencies = [
 Homepage = "https://github.com/cyrraz/plothist"
 Documentation = "https://plothist.readthedocs.io/"
 "Bug Tracker" = "https://github.com/cyrraz/plothist/issues"
+
+[project.scripts]
+plothist_install_fonts = "plothist.scripts.install_latin_modern_fonts:install_fonts"
```
[the remark this answers is not in the document]
Comment (0ctagon) [reply pyproject.toml:32]: This creates a {plothist_install_fonts` command that you can run on your terminal. If launched, it runs the `install_fonts()` function in `plothist/scripts/install_latin_modern_fonts.py`

File: plothist/scripts/install_latin_modern_fonts.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import subprocess
+import pkg_resources
+import os
+import platform
+
+
+def install_fonts() -> None:
+    script_path = pkg_resources.resource_filename(
+        __name__, "install_latin_modern_fonts.sh"
```
[the remark this answers is not in the document]
Comment (0ctagon) [reply plothist/scripts/install_latin_modern_fonts.py:9]: As discussed, I'll leave it to you.

[the remark this answers is not in the document]
Comment (cyrraz) [reply plothist/scripts/install_latin_modern_fonts.py]: Done

**Comment by cyrraz** [comment]
Hi @0ctagon, please have a look at this update. If it is fine with you, let's merge it. Please ask a macOS user in the office to test it.

**Review by cyrraz — APPROVED** [review]

**Comment by 0ctagon** [comment]
You can merge and do the test version in PyPI to check if everything works. If yes, we can put it in v0.8 with the complex 2D plot.

File: plothist/scripts/install_latin_modern_fonts.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            ["rm", "-f", (font_directory / f"latin-modern-{lm}.zip").as_posix()]
+        )
+
+    # Remove font cache
+    try:
+        subprocess.run(
+            ["rm", matplotlib_font_cache],
+            check=True,
+        )
+    except subprocess.CalledProcessError:
```
[the remark this answers is not in the document]
Comment (cyrraz) [reply plothist/scripts/install_latin_modern_fonts.py:57]: I think so

**Review by cyrraz — APPROVED** [review]

**Pull request merged**